### ISSUE TITLE
Can not use CollectionView  due to System.MissingMethodException

### ISSUE THREAD

**Issue by edgarfgp** [Zaid-Ajaj/fabulous-simple-elements#17]
First of all thanks for this amazing library it makes really easy to write the views.

I was trying to use CollectionView and there is a System.MissingMethodException

![Screenshot 2019-12-26 at 17 31 26](https://user-images.githubusercontent.com/31915729/71483401-f778e980-2807-11ea-97af-245e55cecf30.png)

I have created a branch in order to reproduce this https://github.com/edgarfgp/dynamic-ui/compare/repro-bug-collection-view?expand=1

Thanks very much . Merry Christmas

**Comment by Blener** [comment]
Hello @Zaid-Ajaj 
I did some testing and I believe SimpleElements needs a new package with Fabulous latest release.
@edgarfgp problem seems to be incompatible versions of Fabulous between his project (0.51.0) and SimpleElements (0.50.3).

Thank you, and Merry Christmas to all.

**Comment by Zaid-Ajaj** [comment]
Hello @edgarfgp, thanks for the kind words, I am glad to hear you are enjoying this library :smile: 

As @Blener said, these types of exceptions are usually due to the fact that the library versions aren't aligned where you could be using a later version of Fabulous than a version that this library depends upon. Even though technically it should just work fine because later versions are backwards-compatible (for non-major upgrades), sometimes the linker will have issues of matching the right functions when compiling the Xamarin application so to fix the problem I only publish a new version that depends on the latest Fabulous. 

Long story short, please try to update everything `Fabulous` related to latest
 - `Fabulous.SimpleElements` to 0.40.0
 - `Fabulous` to 0.51.0
 - `Fabulous.XamarinForms` to 0.51.0

Afterwards, the problem should be (hopefully) fixed so I will close the issue. Please re-open it if the problem persists.

Thanks again and Merry Christmas to all.
